### DISCUSSION TITLE
✅ 🚀 Feature Request: Ability to choose icon or image for Mega Menu item. - [ED-9763]

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#21602] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team @avivu, @nicholaszein.

Since you said you are working on adding custom icon for the mega menu item, I wanted to make an important feature request before the official release. I hope you consider it.

It would be really great if you could add an image for the mega menu items in addition to the icon. This is very important.

For example, I want to display a dropdown for logged in users in the header like the image below. In that case, I can show them the profile picture and that would be really cool.

![image in mega menu](https://user-images.githubusercontent.com/103376243/224013629-dd0bec0f-2b85-4252-94b7-19df885d7456.png)

**The Powerpack plugin now offers this feature in its icon list widget.**

![image](https://user-images.githubusercontent.com/103376243/224015815-9abae816-3015-44b0-ac25-e114dec717b1.png)

### Describe the solution you'd like

For the mega menu item, in addition to the icon, add the possibility of using an image (dynamic image).

**Comment by nicholaszein** [comment]
Hi @Joliamendi,

thanks for submitting your feature request! 🙏 

Best regards

**Comment by nicholaszein** [comment]
Hey, @Joliamendi!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro **vX3.13.0**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
